pocket/assetlist.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "pocket",
+  "assets": [
+    {
+      "description": "The native token of Pocket Network",
+      "denom_units": [
+        {
+          "denom": "upokt",
+          "exponent": 0
+        },
+        {
+          "denom": "pokt",
+          "exponent": 6
+        }
+      ],
+      "base": "upokt",
+      "name": "POKT",
+      "display": "pokt",
+      "symbol": "POKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.svg"
+      },
+      "keywords": ["rpc", "api"],
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.svg"
+        },
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.png"
+        }
+      ],
+      "coingecko_id": "pocket-network",
```

Comment:
Oh, I didn't realize that this was an existing token. I think what aught to be done is register the asset under Ethereum's assetlist, and then have this one point to that as an additional mintage. But I think we are ok to push this for now.

Reply:
Could we push it as is?

This is an L1 (cosmos chain) that is not EVM compatible (yet) so would rather register it under the Ethereum's asset list once that part is live.

Reply:
Ok, I'd say if you're not going to connect it, and they could have different prices (by not being interoperable), then we should remove the coingecko id here

Reply:
I don't fully understand by what we are referring to in "connect it".

Here is the link to coingetcko: https://www.coingecko.com/en/coins/pocket-network

Happy to update the ethereum assetlist you're referring to, but I'm just not following the reasoning/logic yet.

Would appreciate your help!

Reply:
What I mean by "connect", I'm referring to how Assets can be defined as having a relationship to another asset. In this case that would mean defining something like:
```
"traces": [
  {
    "type": "additional-mintage",
    "counterparty": {
      "chain_name": "ethereum",
      "base_denom": "0x764a726d9ced0433a8d7643335919deb03a9a935"
    },
    "provider": "Pocket Network"
  }
]
```
but this assumes that the ERC20 version is the original, and defining this relationship could only be accepted if the original POKT were actually defined in the [Ethereum assetlist](https://github.com/cosmos/chain-registry/blob/master/_non-cosmos/ethereum/assetlist.json)